tools/eslint-rules/alphabetize-errors.js

```diff
@@ -27,6 +44,32 @@ module.exports = {
           context.report({ node, message });
         }
       },
+
+      [requireInternalErrorsSelector](node) {
+        if (node.id.type !== 'ObjectPattern') {
+          context.report({ node, message: 'Use destructuring to define error codes used in the file' });
+        }
+      },
+
+      [codesSelector](node) {
+        if (node.value.type !== 'ObjectPattern') {
+          context.report({ node, message: 'Use destructuring to define error codes used in the file' });
```

Comment:
What's wrong with importing `codes` and then using `codes.SOME_ERROR`?

Reply:
Correct, that one is just for consistency (as you can see in the change list, destructuring is what happens in most files already)